pkg/chain/celo/tbtc.go

```diff
@@ -451,10 +465,13 @@ func (ta *tbtcApplication) FundingInfo(
 	if err != nil {
 		return nil, err
 	}
+
+	transactionHash, outputIndex := parseUtxoOutpoint(fundingInfo.UtxoOutpoint)
```

Comment:
Let's add an empty line.

```suggestion
	transactionHash, outputIndex := parseUtxoOutpoint(fundingInfo.UtxoOutpoint)
	
```